src/Microsoft.DotNet.ProjectModel/RuntimeOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class RuntimeOptions
```

Comment:
it this change related to the issue this PR aims to address? if not should this be moved to its own PR?